usr/share/rear/pack/GNU/Linux/900_create_initramfs.sh

```diff
@@ -27,21 +28,20 @@
 # ZVM_NAMING      - set in local.conf, if Y then enable naming override
 # ZVM_KERNEL_NAME - keeps track of kernel name in results array
 # ARCH            - override only if ARCH is Linux-s390
-# 
+#
 # initrd name override is handled in 900_create_initramfs.sh
 # kernel name override is handled in 400_guess_kernel.sh
 # kernel name override is handled in 950_copy_result_files.sh
+            
+if test "$ARCH" = "Linux-s390" ; then
```

Comment:
Can you put this into an arch-specific script?

Reply:
I have no good idea how to properly separate
arch-dependent code away into separate scripts
and still keep the code easy to understand, cf.
https://github.com/rear/rear/pull/3233#issuecomment-2145206590